Integrations/integration-Awake_Security.yml

```diff
@@ -0,0 +1,607 @@
+        elif percentile <= 75:
+            # Something doing multiple things out of the
+            # ordinary, worth investigating
+            score = 2
+        else:
+            # Probably bad or at least not compliant with
+            # company policy.  You have to make an effort
+            # to score this high.
+            score = 3
+
+        return { "Vendor": "Awake Security", "Type": indicator_type, "Indicator": lookup_key, "Score": score }
+
+    def lookup(lookup_type, lookup_key):
+        path = "/lookup/" + lookup_type
+
+        request["lookup_key"] = lookup_key
+
+        if "lookback_minutes" in args:
+            request["lookback_minutes"] = int(args["lookback_minutes"])
+        else:
+            request["lookback_minutes"] = 480
+
+        response = requests.post(prefix + path, json=request, headers=headers, verify=verify)
+
+        return response.json()
+
+    def lookupDevice(lookup_key):
+        contents = lookup("device", lookup_key)
+
+        humanReadableFields = [
+            "deviceScore",
+            "deviceName",
+            "deviceType",
+            "os",
+            "osVersion",
+            "commonEmail",
+            "commonUsername",
+            "tags",
+            "recentIP",
+            "activeIP",
+            "nSimilarDevices",
+            "ipCount",
+            "applicationCount",
+            "protocols",
+            "firstSeen",
+            "lastSeen",
+        ]
+
+        if "deviceScore" in contents:
+            dbotScore = toDBotScore("device", contents["deviceScore"], lookup_key)
+        else:
+            dbotScore = None
+
+        humanReadable = displayTable([contents], humanReadableFields)
+
+        returnResults(contents, humanReadable, dbotScore)
+
+    def lookupDomain(lookup_key):
+        contents = lookup("domain", lookup_key)
+
+        humanReadableFields = [
+            "notability",
+            "isAlexaTopOneMillion",
+            "isDGA",
+            "intelSources",
+            "numAssociatedDevices",
+            "numAssociatedActivities",
+            "approxBytesTransferred",
+            "protocols",
+            "firstSeen",
+            "lastSeen",
+        ]
+
+        if "notability" in contents:
+            dbotScore = toDBotScore("domain", contents["notability"], lookup_key)
+        else:
+            dbotScore = None
+
+        humanReadable = displayTable([contents], humanReadableFields)
+
+        returnResults(contents, humanReadable, dbotScore)
+
+    def lookupEmail(lookup_key):
+        contents = lookup("email", lookup_key)
+
+        humanReadableFields = [
+            "notabilityPercentile",
+            "deviceName",
+            "os",
+            "deviceType",
+            "application",
+            "numberSimilarDevices",
+            "numberSessions",
+            "firstSeen",
+            "lastSeen",
+            "duration",
+            "deviceId",
+        ]
+
+        if "notabilityPercentile" in contents:
```

Comment:
i'll check it thrilly when we'll have access, but i think dbotscore 0 might be a better option here